### DISCUSSION TITLE
[Script request] Pi.Alert is now NetAlert X

### DISCUSSION THREAD

**Discussion by diegogyn** [community-scripts/ProxmoxVE#986] [Request script]
### Application Name

NetAlert X

### Website

https://netalertx.com/

### Description

[jokob-sk](https://github.com/jokob-sk) decided to rename the project to 🚀NetAlertX. This will help prevent confusion about which fork someone is using, and differentiate it from the now stale upstream project. With about 1800 or so commits over the stale project, I thought, this project deserved a new name. It will also remove the confusion about only supporting Raspberry Pi's 😵 . 

Get visibility of what's going on on your WIFI/LAN network. Schedule scans for devices, port changes and get alerts if unknown devices or changes are found. Write your own [Plugins](https://github.com/jokob-sk/NetAlertX/tree/main/front/plugins#readme) with auto-generated UI and in-build notification system. Build out and easily maintain your network source of truth (NSoT).

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
duplicate

**Reply by diegogyn** [reply]
Just to clarify, the current script available is using the repository: https://github.com/leiweibau/Pi.Alert/, which is a fork of the old Pi.Alert. 

The original creator has renamed the version and added new features in the repository: https://github.com/jokob-sk/NetAlertX.

Therefore, I think it’s important to include a warning in the current script or keep both versions available.

**Reply by MickLesk** [reply]
Its an duplicate Request!
